app.js

```diff
@@ -1,13 +1,14 @@
 import express from 'express';
 import bodyParser from 'body-parser';
-
+import route from './dummy/route/index';
```

Comment:
Since the file name is `index`, you can leave out the index at the end of this file import. So it becomes `import route from './dummy/route`